apps/content-script/src/components/ShowWordCardWrapper.svelte

```diff
@@ -1,27 +1,37 @@
 <script lang="ts">
-  import { ShowWordCard } from '@parrotly.io/extension-components';
+  import { ShowWordCard, MCQCard } from '@parrotly.io/extension-components';
   import browser from 'webextension-polyfill';
   import { EXTENSION_MESSAGES } from '@parrotly.io/constants';
   import type {
     IRepetitionWord,
     IUserSettings,
     MESSAGE_SHOW_WORD,
     MESSAGE_KNOW_WORD,
+    MESSAGE_MCQ_ANSWER,
     IRepetitionList,
     MESSAGE_PLAY_TEXT,
   } from '@parrotly.io/types';
+  import McqCard from 'libs/extension-components/src/components/MCQCard.svelte';
```

Comment:
1. This is an unused import
2. We shouldn't import with absolute routes as in this case. Always use @parrotly.io/<<something>>